openmetadata-ui/src/main/resources/ui/src/components/common/rich-text-editor/EditorToolBar.ts

```diff
@@ -0,0 +1,45 @@
+import MarkdownIcon from '../../../assets/svg/markdown.svg';
+
+/**
+ * Read more : https://nhn.github.io/tui.editor/latest/tutorial-example15-customizing-toolbar-buttons
+ * @returns HTMLElement for toolbar
+ */
+const markdownButton = (): HTMLButtonElement => {
+  const button = document.createElement('button');
+
+  button.className = 'toastui-editor-toolbar-icons markdown-icon';
+  button.style.backgroundImage = 'none';
+  button.style.margin = '0';
+  button.style.marginTop = '4px';
+  button.innerHTML = `
+  <a
+    href="https://www.markdownguide.org/cheat-sheet/"
+    rel="noreferrer"
+    target="_blank">
+    <img
+      alt="markdown-icon"
+      className="svg-icon"
+      src=${MarkdownIcon} />
+  </a>`;
+
+  return button;
```

Comment:
any reasons for creating a button like this and not using <button> tag directly?

Reply:
Tui editor expect HTMLElement type and if we use tag then it will be as Element type, so we have to create button this way.